lib/common/components/sheets/linum_bottom_sheet.dart

```diff
@@ -12,28 +12,34 @@ class LinumBottomSheet extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Column(
-      children: [
-        AppBar(
-          primary: false,
-          automaticallyImplyLeading: false,
-          title: Text(
-            title,
-            style: Theme.of(context).textTheme.headlineSmall,
-          ),
-          centerTitle: true,
-          actions: [
-            IconButton(
-              icon: const Icon(Icons.close),
-              onPressed: Navigator.of(context).pop,
+    return Padding(
+      padding: EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
```

Comment:
Keyboard interaction is better now, but there is absolutely no padding between the last piece of content on the bottom sheet and the bottom of my phone screen now. Does not look too good and is harder to navigate with a thumb

Reply:
![Screenshot_2024-07-11-22-31-38-60_01759765ed6224c98974a349ca5ff802](https://github.com/user-attachments/assets/4e125815-e549-4be3-9ada-bb8e910580f2)
![Screenshot_2024-07-11-22-31-42-03_017
![Screenshot_2024-07-11-22-31-44-83_01759765ed6224c98974a349ca5ff802](https://github.com/user-attachments/assets/314958ef-2159-4611-88e0-b0d2dc360f45)
59765ed6224c98974a349ca5ff802](https://github.com/user-attachments/assets/b800a4ae-444b-490c-98fe-3dbc1085fd38)